src/react-apps/applications/runtime/src/shared/resources/language/languages.ts

```diff
@@ -0,0 +1,493 @@
+/* tslint:disable:max-line-length */
+
+export const nb = () => (
+  {
```

Comment:
Følgende seksjoner hører ikke til runtime og kan fjernes:
- dashboard
- administration
- handle_merge_conflict
- sync_header
- testing
- deploy_to_test
- ux_editor
- app_publish
- app_release
- app_create_release
- app_deploy
- app_deploy_table
- app_deploy_messages

Reply:
ux_editor is used by some runtime components. Removing all other.